src/components/FormInput/FormInput.tsx

```diff
@@ -17,7 +17,7 @@ export const FormInput = ({ label, helperText, style, ...restProps }: FormInputP
     return (
         <Container style={style}>
             <Label>{label}</Label>
-            <Input {...restProps} />
+            <Input ref={restProps.forwardRef} {...restProps} />
```

Comment:
forwardRef could be destructured from props to avoid setting it twice as "ref" and "forwardRef"